docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+---
+
+services:
+
+  redis:
```

Comment:
I will suggest using a bit more descriptive name 'redis' -> 'redis-standalone'
In case we add additional services later  e.g. redis-cluster, redis-master-slave...